test/python/quantum_info/states/test_measures.py

```diff
@@ -187,7 +187,7 @@ def test_entropy_density_matrix(self):
         for rho in rhos:
             self.assertAlmostEqual(entropy(rho), 1)
             self.assertAlmostEqual(entropy(rho, base=2), 1)
-            self.assertAlmostEqual(entropy(rho, base=np.e), -np.log(0.5))
+            self.assertAlmostEqual(entropy(rho, base=np.e), -1 * np.log(0.5))
```

Comment:
Why was this added?

Reply:
Lint error about applying negative to type which doesn't support negative.